cypress/e2e/cypressActions.cy.js

```diff
@@ -0,0 +1,85 @@
+/// <reference types='cypress' />
+
+describe('Practice form', () => {
+  before(() => {
+    cy.visit('https://demoqa.com/automation-practice-form');
+  });
+
+  it('should register a student filling the required fields', () => {
+    cy.get('[placeholder="First Name"]')
+    .type('Khrystyna');
```

Comment:
you can move all the values you need to type into the variables and just pass them where needed